tests/src/performance/linkbench/getcert.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\dumpbin.exe" /headers %1 | findstr /C:"Certificates Directory
```

Comment:
I can call VCvarsall.bat to setup developer command prompt and then call dumpbin -- but that's pretty much the same. Did you mean I should use something like 
`%VCINSTALLDIR%\bin\amd64\dumpbin.exe` ?